barman/postgres.py

```diff
@@ -507,36 +528,16 @@ def server_txt_version(self):
             )
             return None
 
-    @property
-    def has_pgespresso(self):
-        """
-        Returns true if the `pgespresso` extension is available
-        """
-        try:
-            # pg_extension is only available from Postgres 9.1+
-            if self.server_version < 90100:
-                return False
-            cur = self._cursor()
-            cur.execute(
-                "SELECT count(*) FROM pg_extension WHERE extname = 'pgespresso'"
-            )
-            q_result = cur.fetchone()[0]
-            return q_result > 0
-        except (PostgresConnectionError, psycopg2.Error) as e:
-            _logger.debug(
-                "Error retrieving pgespresso information: %s", force_str(e).strip()
-            )
-            return None
-
     @property
     def is_in_recovery(self):
         """
         Returns true if PostgreSQL server is in recovery mode (hot standby)
         """
         try:
-            # pg_is_in_recovery is only available from Postgres 9.0+
-            if self.server_version < 90000:
-                return False
+            # # pg_is_in_recovery is only available from Postgres 9.0+
+            # # Todo: remove
```

Comment:
Looks like this can just be removed now?

Reply:
indeed